pkg/cosign/verify.go

```diff
@@ -156,13 +163,113 @@ type CheckOpts struct {
 	// IgnoreTlog skip tlog verification
 	IgnoreTlog bool
 
+	// UseSignedTimestamps use signed timestamps if available
+	UseSignedTimestamps bool
```

Comment:
Is this potentially something that could be split out into its own PR, so that each PR has a clear singular focus?

Reply:
That's a good idea for something we can split out... I think this is needed because the existing timestamp verification in cosign doesn't work with TUF/PGI, and is only enforced if you provide a timestamp cert chain to the verification subcommands.